### PR TITLE
Haley Holmes - Researcher Overview Page Edit

### PR DESCRIPTION
**Pull request by brmscheiner**
Haley Holmes edited this page on 01/03/2021. Mostly wording and grammar edits. Did some minor re-arranging of content in the Introduction and Terminology sections.

### CODE REVIEW

File: docs/researchers/researcher_overview.md [hunk 1]
```diff
@@ -5,107 +5,105 @@ sidebar_label: Overview
 slug: /researchers/overview
 ---
 
-Wildbook is a Web-based, multi-user software platform to help researchers collaboratively track individual animals in wildlife populations and estimate population sizes. Each Wildbook installation can be used by multiple researchers for multiple species. 
+Wildbook is a web-based, multi-user software platform to help researchers collaboratively track individual animals in wildlife populations and estimate population sizes. Each Wildbook installation can be used by multiple researchers for multiple species. 
 
-Wildbook includes:
+This introduction covers the most common interactions within the Wildbook platform. It is intended to support Wildbook users and researchers collecting data in the field and to illustrate how to use Wildbook for data storage, curation, searching, and analysis.
```
Comment (TanyaStere42) [inline docs/researchers/researcher_overview.md]: Something to look out for (it wasn't consistent before, either) is to reference A Wildbook platform, not THE Wildbook platform. We have a lot of confusion around the distinction between the different platforms and the product, and this is one of the ways we can prevent that

File: docs/researchers/researcher_overview.md [hunk 2]
```diff
@@ -5,107 +5,105 @@ sidebar_label: Overview
...
 
 *The Wildbook ecosystem of people, servers, and AI.*
 
-This introduction is intended to support Wildbook users collecting data in the field ("researchers") and then loading it for storage, curation, searching, and analysis in Wildbook. It covers the most common interactions with the Wildbook platform.
-
 ## Wildbook Terminology
 
-The following terms are important in Wildbook. In its simplest usage, Wildbook may receive a wildlife picture plus related metadata (e.g., where the photo was taken and when), which gets uploaded by a researcher. From here, a number of terms come up as we work through the process the system and users take to curate this wildlife data. Let’s use this picture as an example.
+In its simplest usage, Wildbook receives a wildlife picture and its related metadata (e.g. where the photo was taken and when), which gets uploaded by a researcher. From here Wildbook uses certain terminology to assist users in curating their wildlife data. To facilitate a better understanding of these terms, we will work through the process the Wildbook system and users take to upload and manage their wildlife data. Let’s use this picture as an example.
```
Comment (TanyaStere42) [inline docs/researchers/researcher_overview.md]: > From here Wildbook uses certain terminology to assist users in curating their wildlife data

This is kind of confusing; we don't use the terms to help people, they are incidental to helping people.

File: docs/researchers/researcher_overview.md [hunk 3]
```diff
@@ -5,107 +5,105 @@ sidebar_label: Overview
...
+Because more than one animal appears at the same time in this example MediaAsset, we want to ensure that the relationship between these two Encounters is well understood (i.e. *These two animals were sighted together.*). Therefore, Wildbook creates a **Sighting**, representing a broader observation of one or more animals. A Sighting allows researchers to include information about social relationships and observed behaviors when multiple animals are sighted together. Unlike Encounters, Sightings are one-to-many; if you interact with five members of a pack, you have one Sighting that is made up of five Encounters, one for each animal. Each Sighting also generates its own web page in Wildbook.
```
Comment (TanyaStere42) [inline docs/researchers/researcher_overview.md]: "also generates" is not technically accurate

File: docs/researchers/researcher_overview.md [hunk 4]
```diff
@@ -5,107 +5,105 @@ sidebar_label: Overview
...
 
-### Menu options
+### Menu Options
 
-Under the **Submit** menu, there are two options: **Report an encounter** and **Bulk Import**. Both options are used to submit data. Submissions include photos and videos plus related metadata, requiring at least date of sighting, location, and species. Additional metadata fields are available. 
+Under the **Submit** menu, there are two options: **Report an Encounter** and **Bulk Import**. Both options are used to submit data. Report an Encounter can be used by anyone, whereas Bulk Import requires a user to log in to the platform and follow the instructions to first upload photos and then their affiliated Excel data spreadsheet. Submissions include photos, videos, and related metadata, requiring at least the date of sighting, location, and species. Additional metadata fields are available.
```
Comment (TanyaStere42) [inline docs/researchers/researcher_overview.md]: Skimmability has been reduced by putting these together. Is there a way to rework so Report an encounter is discussed completely before Bulk Import? Also can we add links to the related pages?
https://docs.wildme.org/docs/researchers/report_encounter
https://docs.wildme.org/docs/researchers/bulk_import

File: docs/researchers/researcher_overview.md [hunk 5]
```diff
@@ -5,107 +5,105 @@ sidebar_label: Overview
...
-- **Individual Search** pulls up several filters that can be leveraged to generate a filtered list of all individuals that the signed-in user has access to. Once the list is generated, additional filtering and sorting can be performed on the **Individual Search Results** page, as well as delving into additional data formats such as matching images and videos, mapped results, and exportable data formats.
-- **Sighting Search** pulls up several filters that can be leveraged to generate a filtered list of all sightings that the signed-in user has access to.
+- **Encounter Search** pulls up several filters that can be leveraged to generate a filtered list of all the Encounters the signed-in user has access to. Once the list is generated, additional filtering and sorting can be performed on the **Encounter Search Results** page, as well as delving into additional data formats such as matching images and videos, mapped results, export formats, and results calendars.
```
Comment (TanyaStere42) [inline docs/researchers/researcher_overview.md]: since calendar per search; should not be plural

**Review by TanyaStere42 — CHANGES_REQUESTED** [review]
Couple of small changes, but otherwise looks good. Mainly want to make clear that skimmability is the biggest focus in these

File: docs/researchers/researcher_overview.md (hunk 2 above)
Comment (haley-holmes) [reply docs/researchers/researcher_overview.md]: Changed to "The following terminology is used by Wildbook programs as users curate their wildlife data" to replace the sentence using the term "assist".

File: docs/researchers/researcher_overview.md (hunk 5 above)
Comment (haley-holmes) [reply docs/researchers/researcher_overview.md]: All fixed!

File: docs/researchers/researcher_overview.md (hunk 4 above)
Comment (haley-holmes) [reply docs/researchers/researcher_overview.md]: I have re-arranged this section for readability and included the two related webpages. I believe I added them correctly as they worked on my computer, but might be worth a double-check!

File: docs/researchers/researcher_overview.md (hunk 3 above)
Comment (haley-holmes) [reply docs/researchers/researcher_overview.md]: Fixed!

File: docs/researchers/researcher_overview.md (hunk 1 above)
Comment (haley-holmes) [reply docs/researchers/researcher_overview.md]: Reviewed the web page to ensure for the correct language.

**Comment by haley-holmes** [comment]
Implemented Tanya S.'s editing comments on 1/6/21.

**Review by TanyaStere42 — APPROVED** [review]
Wonderful, thank you so much!

**Pull request merged**